src/framework/accessibility/iaccessibilitycontroller.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef MU_ACCESSIBILITY_IACCESSIBILITYCONTROLLER_H
+#define MU_ACCESSIBILITY_IACCESSIBILITYCONTROLLER_H
+
+#include "modularity/imoduleexport.h"
+#include "iaccessibility.h"
+
+namespace mu::accessibility {
+class IAccessibilityController : MODULE_EXPORT_INTERFACE
+{
+    INTERFACE_ID(IAccessibilityController)
+public:
+    virtual ~IAccessibilityController() = default;
+
+    virtual void created(IAccessibility* parent, IAccessibility* item) = 0;
+    virtual void destroyed(IAccessibility* item) = 0;
+    virtual void actived(IAccessibility* item, bool isActive) = 0;
```

Comment:
I just realised that we have both "activated" and "focused" methods at the same time, could you explain when it'd be called and what does it mean for an item that it's active/inactive?

Reply:
The answer to this question not fully clear at the moment. At the moment, for example, "panel" may be active, and "control" may be focused. 

I try to do this clarity later